drivers/gles2/rasterizer_gles2.cpp

```diff
@@ -481,6 +481,8 @@ void RasterizerGLES2::make_current() {
 }
 
 void RasterizerGLES2::register_config() {
+	GLOBAL_DEF("rendering/quality/filters/anisotropic_filter_level", 4);
```

Comment:
Now that both GLES2 and GLES3 have this setting, it should be defined in visual_server.cpp here:
https://github.com/godotengine/godot/blob/0b6dd929c677baad3888579c3f237db43f19592c/servers/visual_server.cpp#L2439-L2441

You will need to remove it from rasterizer_gles3.cpp as well

Reply:
Yeah, that code duplication bugged me, too. I've made the change, and since we're now modifying the GLES3 code, I decided to remove the extraneous write to config.use_anisotropic_filter in rasterizer_storage_gles3.cpp while I was at it.